secure_uninstall/README.md

```diff
@@ -0,0 +1,5 @@
+Secure Uninstall
+================
+
+Ask admin password ('admin_passwd' key from config file)
+before to proceed to module uninstallation
```

Comment:
Elsewhere in the interface (database management screens) this is called 'Master Password'. I think this should be used here too to reduce confusion.